test/jellyfish_web/controllers/peer_controller_test.exs

```diff
@@ -25,7 +25,7 @@ defmodule JellyfishWeb.PeerControllerTest do
     test "renders peer when data is valid", %{conn: conn, room_id: room_id} do
       conn = post(conn, ~p"/room/#{room_id}/peer", type: @peer_type)
       response = json_response(conn, :created)
-      assert_response_schema(response, "PeerDetailsResponse", @schema)
```

Comment:
As we discussed this test shouldn't pass :thinking:

Reply:
Some of the `open_api_spex` were missing `required` key, thus weren't being check for validity in the `assert_response_schema, should be good now 😎